### ISSUE TITLE
Job Posting URL should be shown in Job card

### ISSUE THREAD

**Issue by jatin33** [jobtriage/jobtriage#161]
User should be able to click the job posting url in job card.

- [X] I have searched the [issues](https://github.com/jobtriage/jobtriage/issues) of this repository and believe that this is not a duplicate.

## Summary 💡
When user updates job posting url in the job card update form, a link with name "Job Description url" should appear in card using which he can visit the required sute

## Examples 🌈

![Screenshot from 2020-07-10 16-44-46](https://user-images.githubusercontent.com/13780467/87149495-f9028e80-c2cd-11ea-9f6a-3dd0db4ec4d3.jpg)

## Motivation 🔦
Since this information will be accessed by user frequently, it makes sense to give a easy way to access this.

**Comment by jatin33** [comment]
@koushikmohan1996 @hiteshkundal what are your thoughts on this?

**Comment by koushikmohan1996** [comment]
@jatin33  Yep, this will be helpful. But we can display it as "job url" this will not change the meaning and it will be small

**Comment by koushikmohan1996** [comment]
@all-contributors add @jatin33 for bug

**Comment by arjungrylls** [comment]
Add to #22
